src/PointwiseFunctions/GeneralRelativity/Ricci.cpp

```diff
@@ -50,6 +51,25 @@ tnsr::aa<DataType, SpatialDim, Frame, Index> ricci_tensor(
                d_christoffel_2nd_kind);
   return result;
 }
+
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+void ricci_scalar(
+    const gsl::not_null<Scalar<DataType>*> ricci_scalar_result,
+    const tnsr::aa<DataType, SpatialDim, Frame, Index>& ricci_tensor,
+    const tnsr::AA<DataType, SpatialDim, Frame, Index>& inverse_metric) {
+  *ricci_scalar_result = trace(ricci_tensor, inverse_metric);
```

Comment:
Use the return-by-reference version of the `trace` function here:

```suggestion
  trace(ricci_scalar_result, ricci_tensor, inverse_metric);
```